src/style.css

```
@@ -0,0 +1,233 @@
+
+.imgLogIn{
```

se usa camel case cuando programan con javascript, pero cuando se generan estilos lo más recomendable es usar otro estándar por medio del uso de guiones.
```suggestion
.img-login{
```